tests/MoBi.Tests/Core/SBML/CompartmentImporterSpecs.cs

```diff
@@ -103,6 +105,13 @@ public void ContainerCreationTest3()
             containsSizeParam.ShouldBeTrue();
             containsVolumeParam.ShouldBeTrue();
          }
+
+         [Observation]
+         public void VolumeIsSetFromSize()
+         {
+            var parameter = _container.Children.FirstOrDefault(x => x.Name == SBMLConstants.VOLUME) as IParameter;
```

Comment:
same as the other pr 
This could be greatly simplified. `var parameter = _container.Parameter(SBMLConstants.VOLUME)`